packages/core/src/transports/fetch.ts

```diff
@@ -24,7 +29,7 @@ function sendAsFetch(url: string, body: string, isDebugEnabled = false): void {
   }); // the empty callback is required as otherwise the catch will be ignored
 }
 
-export function getFetchTransport(url: string, isDebugEnabled = false): Transport {
+export function getFetchTransport(url: string, isDebugEnabled = false, options: RequestOptions = {}): Transport {
```

Comment:
This transport already has 3 params. Should we create a single `options` parameter and add the current ones as part of that?

I see an issue with the current approach: you may want to pass headers but not the debugging flag

Reply:
done